cmd/di.go

```diff
@@ -173,6 +173,7 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 	nats_discovery.Bootstrap()
 
 	log.Infof("Starting Mysterium Node (%s)", metadata.VersionAsString())
+	log.Infof("Build information (%s)", metadata.BuildAsString())
```

Comment:
Maybe remake `VersionAsString()` function, so that that we would have build info in all places